src/geojsonm/geojsonm.mli

```diff
@@ -79,3 +79,6 @@ val fold_geometry :
 val fold_props :
   ('a -> Ezjsonm.value -> 'a) -> 'a -> Jsonm.src -> ('a, Err.t) result
 (** [fold_props f init src] *)
+
+val iter_geometry : (G.Geometry.t -> unit) -> Jsonm.src -> (unit, Err.t) result
```

Comment:
Some documentation in the same style as `map_*` and `fold_*`  would be great here.

Reply:
Yes Sir, @patricoferris done now,